documentacion/posintegrado/README.md

```diff
@@ -0,0 +1,275 @@
+puedes identificar usando la función mencionada en el apartado anterior #Listar puertos disponibles). También
+necesitaras el baudrate al cual esta configurado el puerto serial del POS Integrado (Por defecto es 115200),
+y puedes obtener los distintos valores desde la clase `TbkBaudrates` del paquete `Transbank.POS.Utils`.
+
+Si el puerto no puede ser abierto, se lanzara una exception `TransbankException`.
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS;
+using Transbank.POS.Utils;
+//...
+string portName = "COM3";
+POS.Instance.OpenPort(portName, TbkBaudrate.TBK_115200);
+```
+
+```c
+#include "transbank.h"
+#include "transbank_serial_utils.h"
+//...
+char* portName = "COM4";
+int retval = open_port(portName, 115200);
+if ( retval == TBK_OK ){
+    //...
+}
+```
+
+### Cerrar un puerto Serial
+
+Al finalizar el uso del POS, o si se desea desconectar de la Caja se debe liberar el puerto serial abierto anteriormente.
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS;
+//...
+bool closed = POS.Instance.ClosePort();
+```
+
+```c
+#include "transbank.h"
+#include "transbank_serial_utils.h"
+//...
+retval = close_port();
+if(retval == SP_OK){
+    //...
+}
+```
+
+### Transacción de Venta
+
+Este comando es enviado por la caja para solicitar la ejecución de una venta. Los siguientes parámetros deben ser enviados desde la caja:
+
+- `Monto`: Monto en pesos informados al POS. Este parámetro es remitido a Transbank para realizar la autorización.
+- `Número Ticket/Boleta`: Este número es impreso por el POS en el voucher que se genera luego de la venta.
+- `Enviar Mensaje`: Este parámetro indica al POS si debe enviar mensajes intermedios a la caja mientras se realiza el proceso de venta. Los mensajes intermedios que envía el POS y que deben ser mostrados por la Caja, deben corresponder según los siguientes códigos:
+  - `78`: Lectura de Tarjeta.
+  - `79`: Confirmación de Monto.
+  - `80`: Selección de Cuotas.
+  - `81`: Ingreso de Pinpass.
+  - `82`: Envío de transacción a Transbank.
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS;
+using Transbank.POS.Responses;
+//...
+SaleResponse response = POS.Instance.Sale(ammount, ticket);
+```
+
+```c
+#include "transbank.h"
+#include "transbank_serial_utils.h"
+//...
+char* response = sale(ammount, ticket, false);
+```
+
+### Transacción de Cierre
+
+Este comando es gatillado por la caja y no recibe parámetros. El POS ejecuta la transacción de cierre contra el Autorizador (no se contempla Batch Upload). Como respuesta el POS Integrado enviará un aprobado o rechazado. (Puedes ver la tabla de respuestas en este [link](/referencias/posintegrado#tabla-de-respuestas))
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS;
+using Transbank.POS.Responses;
+//...
+RegisterCloseResponse response = POS.Instance.RegisterClose();
```

Comment:
Is "RegisterClose" the more natural name for this transaction? Maybe `CloseRegister` or just `Close` might work better? I don't really know, but I recommend giving it a thought before 1.0

Reply:
I will change it to Close, i doubted at first, because it can be confused with the port close action.